.gitlab-ci.yml

```diff
@@ -70,38 +57,3 @@ release:
     paths:
       - VERSION.txt
     expire_in: 1 week
-
```

Comment:
Did this get moved into one of the templates or no longer used?

Reply:
https://github.com/NCI-GDC/gitlab-templates/blob/master/templates/common/docker.yaml